cmd/ui/src/utils.ts

```diff
@@ -77,7 +77,8 @@ export const initializeBHEClient = () => {
                 if (
                     error?.response?.status === 401 &&
                     error?.response?.config.url !== '/api/v2/login' &&
-                    error?.response?.config.url !== '/api/v2/logout'
+                    error?.response?.config.url !== '/api/v2/logout' &&
+                    !error?.response?.config.url.match('/api/v2/bloodhound-users/[a-z0-9-]+/secret')
```

Comment:
I tried a few different variants such as `e.preventDefault()` or `e.stopPropagation()` but no joy. Ended up opting for this which fixed the issue of immediate logout upon an invalid current password. Alternatively I could use a different code but `401` and `403` have built in default behavior and yet `401` felt the most appropriate error code to return from API when an invalid current password is sent. 

Of course, I'm open to suggestions if there's a cleaner way of reporting invalid current password!

Reply:
See previous comment considering a 403 status instead.